src/test/kotlin/no/nav/helsearbeidsgiver/bro/sykepenger/testutils/MockUtils.kt

```diff
@@ -45,8 +46,8 @@ fun mockForespoerselDto(): ForespoerselDto =
         forespoerselId = randomUuid(),
         type = Type.KOMPLETT,
         status = Status.AKTIV,
-        orgnr = "123456789".let(::Orgnr),
-        fnr = "11223344556",
+        orgnr = randomDigitString(9).let(::Orgnr),
+        fnr = randomDigitString(11),
```

Comment:
Et lite forsøk på å skape mer unike testdata. Denne mock-funksjonen er tungt brukt for i `ForespoerselDatTest`, så vil jeg unngå at vi passerer en likhetssjekk simpelthen fordi mock-objektene er for like.